robot-core/src/main/java/org/obolibrary/robot/MaterializeOperation.java

```diff
@@ -143,7 +169,7 @@ public static void materialize(
         logger.debug("Excluding classes not in main ontology: " + c);
         continue;
       }
-      Set<OWLClassExpression> sces = emr.getSuperClassExpressions(c, true);
+      Set<OWLClassExpression> sces = emr.getSuperClassExpressions(c, !includeIndirect);
```

Comment:
Ok this _seems_ to make sense! Needs someone from gene ontology reasoner time to check.. @balhoff @cmungall